INSTALL.md

```diff
@@ -14,4 +14,75 @@ You can run tests with:
 cargo test
 ```
 
-Please refer to the cargo documentation for more detailed instructions.
\ No newline at end of file
+Please refer to the cargo documentation for more detailed instructions.
+
+
+### If you want to create a system service to run your application follow the instructions below:
+
+
+# Systemd setup for backend
+
+
+#### Create a simple file copying the contents below to the file, and use the command:
+`sudo nano /etc/systemd/system/teosd.service`
+#### This will create a file named teos.service in the /etc/systemd/system/ path
+
+#### One you create it and copy and paste, change the information with your proper installation path, user, and comments:
+
```

Comment:
Copy the following information replacing the relevant information with yours (`<USER>` and `<GROUP>`):

Reply:
Done